src/super_gradients/examples/predict/detection_predict_with_labels.py

```diff
@@ -0,0 +1,27 @@
+from super_gradients.common.object_names import Models
```

Comment:
Do you want to convert this script to ipynb fine and generate a documenation from it automatically similary to what I did for model export?

Reply:
sure, how does one do that ?

Reply:
Actually thinking it over again, perhaps what in here is a bit dull for a notebook.
Maybe we should unite our predict examples to one notebook, WDYT ?

Reply:
I think having a single predict notebook would be a great way to show how one can utilize predict() for different tasks.
What you want to have to achieve this is a notebook file somewhere in `super_gradients/examples` and a modified Makefile target:

```Makefile
examples_to_docs:
	jupyter nbconvert --to markdown --output-dir="documentation/source/" --execute src/super_gradients/examples/model_export/models_export.ipynb
	jupyter nbconvert --to markdown --output-dir="documentation/source/" --execute src/super_gradients/examples/using_models_predict/using_models_predict.ipynb
```

This way each time one executes `examples_to_docs` target a documentation is generated from the output of the notebook.